v3/fleet-local/control/booster-metrics-producer@.service

```diff
@@ -0,0 +1,34 @@
+[Unit]
+Description=Booster Metrics Collector @ %i
+Requires=docker.service
+After=docker.service flight-director@i.service
+
+[Service]
+User=core
+Restart=on-failure
+
+ExecStartPre=/usr/bin/sh -c "/usr/bin/docker pull $(etcdctl get /images/booster/metrics-producer)"
+ExecStartPre=-/usr/bin/docker kill booster
+ExecStartPre=-/usr/bin/docker rm booster
+ExecStart=/usr/bin/sh -c "/usr/bin/docker run \
+  --name booster \
+  --net=host \
+  -e AWS_DEFAULT_REGION=us-east-1 \
```

Comment:
Like this?

-e AWS_DEFAULT_REGION=$(curl -s http://169.254.169.254/latest/dynamic/instance-identity/document|jq .region)